tests/templates/kuttl/external-access/40-assert.yaml

```diff
@@ -0,0 +1,114 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: test-available-condition
+timeout: 600
+commands:
+  - script: kubectl -n $NAMESPACE wait --for=condition=available airflowclusters.airflow.stackable.tech/airflow --timeout 301s
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+metadata:
+  name: test-airflow-cluster
+timeout: 1200
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: airflow-webserver-default
+spec:
+  template:
+    spec:
+      terminationGracePeriodSeconds: 120
+status:
+  readyReplicas: 2
+  replicas: 2
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: airflow-webserver-external-unstable
+status:
+  readyReplicas: 1
+  replicas: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: airflow-webserver-cluster-internal
+status:
+  readyReplicas: 1
+  replicas: 1
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: airflow-worker-default
+spec:
+  template:
+    spec:
+      terminationGracePeriodSeconds: 300
+status:
+  readyReplicas: 2
+  replicas: 2
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: airflow-scheduler-default
+spec:
+  template:
+    spec:
+      terminationGracePeriodSeconds: 120
+status:
+  readyReplicas: 1
+  replicas: 1
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: airflow-webserver
+status:
+  expectedPods: 4
+  currentHealthy: 4
+  disruptionsAllowed: 1
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: airflow-worker
+status:
+  expectedPods: 2
+  currentHealthy: 2
+  disruptionsAllowed: 1
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: airflow-scheduler
+status:
+  expectedPods: 1
+  currentHealthy: 1
+  disruptionsAllowed: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: airflow-webserver-cluster-internal-group
+spec:
+  type: ClusterIP # cluster-internal
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: airflow-webserver-default-group
+spec:
+  type: NodePort # external-stable
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: airflow-webserver-external-unstable-group
+spec:
+  type: NodePort # external-unstable
```

Comment:
Hm, do we want to install test-specific ListenerClasses? Probably don't want to fail over a cluster being installed with the cloud preset instead.

Reply:
I'm not sure I understand: I'm using the standard listener classes and doing a trivial test for each one.

Reply:
Yeah, in particular you can't *rely* on that `external-stable` always creates a NodePort. For example: https://docs.stackable.tech/home/stable/listener-operator/listenerclass/#preset-ephemeral-nodes configures it to use LoadBalancer instead.

Reply:
Ah, I see. Hmm.

Reply:
I've added custom listener classes here to stay independent of any future changes: 57e1644

Reply:
:+1:

As per convention, those classes should include the name of the test namespace to prevent clashes when running multiple tests in parallel.

Reply:
Fixed ea88ae4